FhirToDataLake/test/Microsoft.Health.Fhir.Synapse.Core.UnitTests/Jobs/FhirToDataLakeAzureStorageJobInfoTests.cs

```diff
@@ -159,5 +161,38 @@ public void GivenTwoSameDefinitionsForJobVersionV1AndV2_WhenGetJobIdentifier_The
             };
             Assert.NotEqual(jobInfo1.JobIdentifier(), jobInfo2.JobIdentifier());
         }
+
+        [Fact]
+        public void
+            GivenJobVersionV1JobDefinition_WhenGetJobIdentifier_ThenTheJobVersionShouldBeRemoved()
+        {
+            var orchestratorDefinition = new FhirToDataLakeOrchestratorJobInputData
+            {
+                JobType = JobType.Orchestrator,
+                JobVersion = SupportedJobVersion.V1,
+                TriggerSequenceId = 1,
+                Since = DateTimeOffset.MinValue,
+                DataStartTime = DateTimeOffset.MinValue,
+                DataEndTime = DateTimeOffset.UtcNow,
+            };
+
+            var jobInfo = new FhirToDataLakeAzureStorageJobInfo()
+            {
+                Id = 1L,
+                QueueType = 0,
+                Status = JobStatus.Created,
+                GroupId = 0,
+                Definition = JsonConvert.SerializeObject(orchestratorDefinition),
+                Result = string.Empty,
+                CancelRequested = false,
+                CreateDate = DateTime.UtcNow,
+                HeartbeatDateTime = DateTime.UtcNow,
+            };
+
+            var jobject = JObject.FromObject(orchestratorDefinition);
+            jobject[JobVersionManager.JobVersionKey].Parent.Remove();
+
+            Assert.Equal(jobject.ToString().ComputeHash(),jobInfo.JobIdentifier());
```

Comment:
We can also add Assert.NotEqual to directly compute hash

Reply:
Added.